scripts/k8s-deploy-forklift.sh

```diff
@@ -11,21 +14,59 @@ while ! kubectl get deployment -n olm olm-operator; do sleep 10; done
 kubectl wait deployment -n olm olm-operator --for condition=Available=True --timeout=180s
 
 # Install forklift-operator
-export FORK_RELEASE=$(curl -s https://api.github.com/repos/konveyor/forklift-operator/branches | jq ". | last .name" | tr -d '"')
-kubectl apply -f https://raw.githubusercontent.com/konveyor/forklift-operator/${FORK_RELEASE}/forklift-k8s.yaml
+
+cat << EOF | kubectl -n ${FORKLIFT_NAMESPACE} apply -f -
```

Comment:
Other than the potential for a broken URL, is there any particular advantage to having the install yaml here instead of using the github version?

Reply:
using one from github is best, but the one "forkliftci" is building the image it'self :-(, we need the one from quay
https://github.com/kubev2v/forkliftci/blob/main/forklift-k8s.yaml#L16

we can use the one from "forklift-operator", but it will be deprected soon, once we move to use "forklift"

another option is to push a PR to forkliftci that allow using any image, but I like it better to have the UI scripts where we can tweak them when we need to

Reply:
pros:
a. better "broken url" protection
b. can tweak namespace and image version

cons:
a. managing our own scripts instead of using forkliftci